### PR TITLE
creating the user interface

### PR DESCRIPTION
**Pull request by the-dunny**
worked on a switch statement for the user interface

### CODE REVIEW

File: src/Driver.java [hunk 1]
```diff
@@ -7,7 +9,78 @@ public static void main(String[] args) {
 		// TODO Write Code
 		System.out.println("Running...");
 		System.out.println("Done.");
+		UserInterface();
+
 		return;
 	}
 
+	public static void UserInterface() {
```
Comment (jackhenry) [inline src/Driver.java]: Method names should follow camel case. Also, method names are typically in verb form. Perhaps the method could be "presentUserInterface()" or "showUserInterface()"?

File: src/Driver.java [hunk 2]
```diff
@@ -7,7 +9,78 @@ public static void main(String[] args) {
...
+				+ "4) Check out a member's items\n5) Process a shipment\n"
+				+ "6) Change the price of a product\n7) Retrieve product info"
+				+ "\n8) Retrieve member info\n9) Print transactions\n"
+				+ "10) List all outstanding orders\n11) List all members with member info\n"
+				+ "12) List all products with product info\n13) Save\n14) Help \n"
+				+ "Press 0 at any time to quit the application";
+		System.out.println(menu);
+		boolean continueApplication = true;
+		while( continueApplication ) {
+			int userInput = scanner.nextInt();
```
Comment (jackhenry) [inline src/Driver.java]: Add a prompt for the user.

`System.out.print("Enter a command: ")`

Using `print()` instead of `println()` will look cleaner.

**Review by jackhenry — COMMENTED** [review]
Check notes

File: src/Driver.java (hunk 1 above)
Comment (the-dunny) [reply src/Driver.java]: right right.  either is fine with me.

File: src/Driver.java (hunk 2 above)
Comment (the-dunny) [reply src/Driver.java]: sounds good I'll straighten it out.

File: ICS372_Dodd_gp1/src/Driver.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+				+ "1) Enroll a member\n2) Remove a member\n3) Add a product\n"
+				+ "4) Check out a member's items\n5) Process a shipment\n"
+				+ "6) Change the price of a product\n7) Retrieve product info"
+				+ "\n8) Retrieve member info\n9) Print transactions\n"
+				+ "10) List all outstanding orders\n11) List all members with member info\n"
+				+ "12) List all products with product info\n13) Save\n14) Help \n"
+				+ "Press 0 at any time to quit the application";
+		System.out.println(menu);
+		boolean continueApplication = true;
+		while( continueApplication ) {
```
Comment (jackhenry) [inline ICS372_Dodd_gp1/src/Driver.java]: According to the professor's coding standards, there should be no spaces after left parentheses or before right parentheses.

File: ICS372_Dodd_gp1/src/Driver.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+			System.out.print("Enter: ");
+			int userInput;
+			try {
+				userInput = scanner.nextInt();
+				switch( userInput ) {
```
Comment (jackhenry) [inline ICS372_Dodd_gp1/src/Driver.java]: According to the professor's coding standards, there should be no spaces after left parentheses or before right parentheses.

**Comment by jackhenry** [comment]
@the-dunny You accidentally added another source set to the eclipse project. I removed it and fixed the remaining formatting issues. Overall, it seems like you got the hang of things. I'm going to merge your code.

**Pull request merged**